paper-textarea.html

```diff
@@ -96,18 +96,18 @@
 
     behaviors: [
       Polymer.PaperInputBehavior,
-      Polymer.IronFormElementBehavior
+      Polymer.IronFormElementBehavior,
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma

Can revert it if you prefer